arangod/Replication/DatabaseInitialSyncer.cpp

```diff
@@ -655,7 +655,7 @@ Result DatabaseInitialSyncer::parseCollectionDump(transaction::Methods& trx,
       } catch (velocypack::Exception const& e) {
         LOG_TOPIC("746ea", ERR, Logger::REPLICATION)
             << "while parsing collection dump: " << e.what();
-        return Result(e.errorCode(), e.what());
+        return Result(TRI_ERROR_INTERNAL, e.what());
```

Comment:
I think we should use `TRI_ERROR_HTTP_CORRUPTED_JSON` here. That seems to be the closest error code we have, and we use it in other places when there are JSON parse errors.

Reply:
That's better, changed it here and above.